sdk/identity/identity/src/credentials/clientCertificateCredentialOptions.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TokenCredentialOptions } from "../client/identityClient";
+
+/**
+ * Defines options for the SubjectNameAndIssuerCredential class.
+ */
+export interface ClientCertificateCredentialOptions extends TokenCredentialOptions {
+
+  /**
+   * Option to include x5c header for SubjectName and Issuer name authorization
+   * Set this option to send base64 encoded public certificate in the client assertion header as an x5c claim
```

Comment:
I wonder if it would be better to merge these two lines into one. Ignore this comment if it seems like it wouldn't make it better.

Reply:
The two comment lines?

Reply:
Yes yes! Perhaps I should have said "nit" 😋

Reply:
It's probably okay as is. I'll put a period on the first sentence though, to make them separate.

Reply:
Alright! Sounds good to me! I'll resolve this.